hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -1,4 +1,4 @@
-#define DRV_VER "@DO_NOT_CHANGE@"
+#define DRV_VER "v0.7-41-g8704a01-dirty"
```

Comment:
huh, sneaky! I'll get rid of this once we're getting ready to merge

Reply:
Yeah, I'm going to fix this in the next release cycle... The build processes modifies files in-place. It should not do this at all.